hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/VolumeInfo.java

```diff
@@ -83,6 +88,24 @@ public VolumeInfo build() throws IOException {
     }
   }
 
+  private long getReserved(ConfigurationSource conf) {
+    Collection<String> reserveList = conf.getTrimmedStringCollection(
+        HDDS_DATANODE_DIR_DU_RESERVED);
+    for (String reserve : reserveList) {
+      String[] words = reserve.split(":");
+      if (words.length < 2) {
+        throw new IllegalArgumentException(
+            "Reserved space should config in pair");
+      }
+
+      if (words[0].trim().equals(rootDir)) {
+        return Long.parseLong(words[1].trim());
```

Comment:
This throws exception if the specified value is not a number.  Also, what about negative values?

Now reserved space needs to be bytes.  Can we add support for storage units (eg. KB, MB, etc.)?  `StorageSize.parse` may be helpful here.